package.json

```diff
@@ -94,11 +94,13 @@
     "boom": "^5.2.0",
     "cids": "^0.5.1",
     "debug": "^3.0.0",
+    "fnv1a": "^1.0.1",
     "fsm-event": "^2.1.0",
     "glob": "^7.1.2",
     "hapi": "^16.5.2",
     "hapi-set-header": "^1.0.2",
     "hoek": "^4.2.0",
+    "interface-datastore": "^0.3.0",
```

Comment:
Was this necessary?

Reply:
Got it.

@dignifiedquire is there any reason why the .get and .put couldn't take a string or a buffer? I understand that `Key` has some utility methods, but the cast could happen internally as well.